storage.bs

```diff
@@ -471,20 +471,26 @@ locally.
 
 <h2 id=usage-and-quota>Usage and quota</h2>
 
-<p>The <dfn export>storage usage</dfn> of a <a>storage shelf</a> is a rough estimate of the amount
-of bytes used by it.
+<p>The <dfn export>storage usage</dfn> of a <a>storage shelf</a> is an <a>implementation-defined</a>
+rough estimate of the amount of bytes used by it.
 
 <p class=note>This cannot be an exact amount as user agents might, and are encouraged to, use
 deduplication, compression, and other techniques that obscure exactly how much bytes a
 <a>storage shelf</a> uses.
 
-<p>The <dfn export>storage quota</dfn> of a <a>storage shelf</a> is a conservative estimate of the
-total amount of bytes it can hold. This amount should be less than the total available storage space
-on the device to give users some wiggle room.
+<p tracking-vector>The <dfn export>storage quota</dfn> of a <a>storage shelf</a> is an
+<a>implementation-defined</a> conservative estimate of the total amount of bytes it can hold. This
+amount should be less than the total storage space on the device. It must not be a function of the
+available storage space on the device.
 
-<p class=note>User agents are strongly encouraged to consider navigation frequency, recency of
-visits, bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} when
-evaluating quotas.
+<div class=note>
+ <p>User agents are strongly encouraged to consider navigation frequency, recency of visits,
```

Comment:
Putting "strong encouragements" in non-normative notes instead of as "shoulds" is a bit weird.

Reply:
Well, these are suggestions. They are not really directly observable so it seems weird to require them.